backend/FwHeadless/Services/CombiningProgress.cs

```diff
@@ -0,0 +1,71 @@
+using SIL.Progress;
+
+namespace FwHeadless.Services;
+
+public class CombiningProgress(params IEnumerable<IProgress?> progresses) : IProgress
```

Comment:
let's use `ICollection<T>` instead. When  consuming an `IEnumerable` it's not safe to assume that you can iterate it more than once